src/libraries/Builder/Microsoft.Agents.Builder/IStreamingResponse.cs

```diff
@@ -60,12 +60,31 @@ public interface IStreamingResponse
         /// </summary>
         List<ClientCitation>? Citations { get; }
 
+        /// <summary>
+        /// Adds a citation for the response.
+        /// </summary>
+        /// <param name="citation"></param>
+        void AddCitation(ClientCitation citation);
+
+        /// <summary>
+        /// Adds a citation for the response.
+        /// </summary>
+        /// <param name="citation"></param>
+        /// <param name="citationPosition"></param>
+        void AddCitation(Citation citation, int citationPosition);
+
         /// <summary>
         ///  Sets the citations for the full message.
         /// </summary>
         /// <param name="citations">Citations to be included in the message.</param>
         void AddCitations(IList<Citation> citations);
```

Comment:
Should we obsolete these `Citation` variations?

Reply:
there is some discussion about allowing general URL links for images that would only work that way. 
Im waiting to see how that pans out.